apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -6087,9 +6087,6 @@ highcharts-chart {
 .panel {
     border-radius: 2px;
 }
-.invoice-modal-content .app-aside-menu-other-taxes:host {
```

Comment:
Please check this class is getting removed

Reply:
checked